.mailmap

```diff
@@ -182,8 +182,9 @@ Boyd Gerber <gerberb@zenez.com> 0000-Admin (0000) <root@devsys0.zenez.com>
 Bram <perl-rt@wizbit.be> Bram (via RT) <perlbug-followup@perl.org>
 Bram <perl-rt@wizbit.be> Bram <p5p@perl.wizbit.be>
 Bram <perl-rt@wizbit.be> Bram via RT <perlbug-comment@perl.org>
-brian d foy <brian.d.foy@gmail.com> brian d foy <bdfoy@cpan.org>
-brian d foy <brian.d.foy@gmail.com> brian d foy <unknown>
+brian d foy <briandfoy@pobox.com> brian d foy <bdfoy@cpan.org>
+brian d foy <briandfoy@pobox.com> brian d foy <briandfoy@cpan.org>
+brian d foy <briandfoy@pobox.com> brian d foy <brian.d.foy@gmail.com>
```

Comment:
`AUTHORS` also needs updating. The esiest way to change the primary email addres is to run
 
```
./Porting/updateAUTHORS.pl --change-email brian.d.foy@gmail.com=briandfoy@pobox.com
```

Reply:
Depends on what you're trying to do. My understanding is that `AUTHORS` is the canonical list of names/addresses and `.mailmap` contains aliases or alternative names/addresses that should be mapped to their canonical form for display purposes.

So if you want to send a patch from a different email address that should still be attributed to you, you just add a line to `.mailmap`. But if you want to change your canonical address, you need to update `AUTHORS` (and `.mailmap` too).